test/xpu_api/utilities/pairs/pair.astuple/tuple_element_pair_as_tuple.pass.cpp

```diff
@@ -0,0 +1,104 @@
+// -*- C++ -*-
```

Comment:
The name is strange. I propose somehing like `tuple_element_pair.pass.cpp`

Reply:
Done